Sample-App/app/src/main/java/com/adobe/marketing/mobile/sampleapp/EdgeTab.java

```diff
@@ -135,6 +143,78 @@ public void onClick(View v) {
         // Setup product list spinner
         Spinner spinner = view.findViewById(R.id.products_spinner);
         spinner.setAdapter(new SpinnerAdapter(getActivity(), android.R.layout.simple_spinner_item, products));
+
+
+        // EdgeIdentity API's
```

Comment:
this looks a little crowded in a single page, can we create separate pages for Edge Identity and Consent samples?

Reply:
Also, can we remove the top tabs if we are using buttons on main page?

Reply:
Created separate pages for Consent and IdentityEdge with textView to show the latest data.

We can still keep the tab... trust me is super easy to navigate once your in that page

Reply:
👍  for now it should be ok, but for the future it does not accommodate multiple extensions